src/main/java/com/flowingcode/vaadin/addons/twincolgrid/TwinColGrid.java

```diff
@@ -66,6 +66,7 @@
 
 @SuppressWarnings("serial")
 @CssImport(value = "./styles/multiselect-cb-hide.css", themeFor = "vaadin-grid")
+@CssImport(value = "./styles/actions-buttons.css", themeFor = "vaadin-button")

```

Comment:
This can be a global style. There is no need to target the shadow DOM just for setting the `:host`width.